docs/sources/mimir/release-notes/v2.10.md

```diff
@@ -0,0 +1,106 @@
+- **Improved the performance of the OTLP ingestion** and more detailed information was added to the traces in order to make troubleshooting problems easier.
+- **Improved the performance of series matching in the store-gateway** by always including the `__name__` posting group causing a reduction in the number of object storage API calls.
+- **Improved the performance of _label values with matchers_** calls when number of matched series is small. If you're using Grafana to query Grafana Mimir make sure you configure the right version in your Prometheus datasource configuration.
+- **Support to cache cardinality, label names and label values** query responses in query frontend. The cache will be used when `-query-frontend.cache-results` is enabled, and `-query-frontend.results-cache-ttl-for-cardinality-query` or `-query-frontend.results-cache-ttl-for-labels-query` set to a value greater than 0.
+- **Reduce wasted effort spent computing results that won't be used** by making queriers cancel the requests sent to a zone upon receiving first error from that zone.
+- **Added the ability to clean up the object storaage by the compactor** when no blocks left in the bucket index by removing the bucket index, markers and debug files. This cleanup can be enabled by using `-compactor.no-blocks-file-cleanup-enabled` option.
+- **Added new debug HTTP endpoints** `/ingester/tenants` and `/ingester/tsdb/{tenant}` to the ingester that provide debug information about tenants and their TSDBs.
+
+Additionally, the following previously experimental features are now considered stable:
+
+- **Support for a ruler storage cache**. This cache should reduce the number of "list objects" API calls issued to the object storage when there are 2+ ruler replicas running in a Mimir cluster. The cache can be configured by setting the `-ruler-storage.cache.*` CLI flags or their respective YAML config options.
+- **Query sharding cardinality estimation.** You can enable it through the advanced CLI configuration flag `-query-frontend.query-sharding-target-series-per-shard`, allows query sharding to take into account cardinality of similar requests executed previously when computing the maximum number of shards to use. If you want to try it out, we recommend starting with a value of `2500`.
+- **Query expression size limit**. You can limit the size in bytes of the queries allowed to be processed through the CLI configuration flag `-query-frontend.max-query-expression-size-bytes`.
+- **Peer discovery / tenant sharding for overrides exporters**. You can enable it through the CLI configuration flag `-overrides-exporter.ring.enabled`.
+- **Overrides exporter enabled metrics selection**. You can select which metrics the overrides exporter should export through the CLI configuration flag `-overrides-exporter.enabled-metrics`.
+- **Per-tenant results cache TTL**.
```

Comment:
Do you want to mention the YAML parameters for this too?

Reply:
Mentioned.